### ISSUE TITLE
Feature request: Hotkeys for more functions

### ISSUE THREAD

**Issue by dominicev** [AliView/AliView#98]
Hey! I love Aliview. As soon as I downloaded it immediately replaced Mesquite as my goto alignment editor/viewer.

It would be so great if the "Delete gap only characters" and "Delete empty sequences" and "Sort by selected character column" options had easy short-cuts or hotkeys. I use those a lot (as I am sure many people do) and it would be time-saving to have a shortcut.

**Comment by andersla** [comment]
Thanks for your kind words about AliView!
I will try to include shortcut-keys in next version

**Comment by JGLAHE** [comment]
I'm merging my issue/feature-request with this one, but I would like to further specify: could we get a menu option for customization of hotkeys? Thank you very much

**Comment by salamandrid** [comment]
I completely agree, I would very much like to have keyboard shortcuts for exactly these items!

**Comment by JGLAHE** [comment]
@salamandrid If you're using Windows 10 I've come up with a temporary fix [https://github.com/AliView/AliView/issues/110]
